src/openms/thirdparty/hdf5-blosc-1.0.0/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.10)
+project(blosc_hdf5)
+include(ExternalProject)
+
+set(BLOSC_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/blosc")
+set(BLOSC_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/blosc")
+set(BLOSC_CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${BLOSC_INSTALL_DIR})
+
+message("BLOSC_PREFIX='${BLOSC_PREFIX}'")
+message("BLOSC_INSTALL_DIR='${BLOSC_INSTALL_DIR}'")
+message("BLOSC_CMAKE_ARGS='${BLOSC_CMAKE_ARGS}'")
+
+ExternalProject_Add(project_blosc
+  PREFIX ${BLOSC_PREFIX}
+  GIT_TAG v1.21.3
+  GIT_REPOSITORY https://github.com/Blosc/c-blosc.git    
+  INSTALL_DIR ${BLOSC_INSTALL_DIR}
+  CMAKE_ARGS ${BLOSC_CMAKE_ARGS}
+)
+
+# sources
+set(SOURCES src/blosc_filter.c)
+set(PLUGIN_SOURCES src/blosc_filter.c src/blosc_plugin.c )
+
+# dependencies
+if(MSVC)
+    # FindHDF5.cmake does not find Windows installations. Try to
+    # use an environment variable instead until the official "find"
+    # file can be updated for Windows.
+    #
+    # Note that you have to set this environment variable by hand.
+    file(TO_CMAKE_PATH "$ENV{HDF5_DIR}" HDF5_HINT)
+    set(HDF5_DIR ${HDF5_HINT} CACHE STRING "Path to HDF5 CMake config directory.")
+    find_package(HDF5 REQUIRED HINTS ${HDF5_DIR})
+else(MSVC)
+    find_package(HDF5 REQUIRED)
+endif(MSVC)
+include_directories(${HDF5_INCLUDE_DIRS})
```

Comment:
not needed. probably their configuration was wrong.
instead of include_directories, use target_link_libraries with the target.

Reply:
Try without find_package. Should be found already in our parent CMake. Or do if(NOT TARGET ...) find_package again.